### ISSUE TITLE
Cache clear try to clear on the wrong directory de_

### ISSUE THREAD

**Issue by Soullivaneuh** [symfony/symfony#14353]
Since I updated symfony to `2.7.0-BETA1`, I have the following error by running `php app/console cache:clear`:

```sh
[...]
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470dfc458.25890768idwnXG): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e043e0.09461100RnOxxV): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e115f2.06201572bHRp89): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e19672.1750055300JIJo): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e2c169.55453635fp31lD): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e34599.66210343cLWMYR): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e3c5e3.66862762eLHYB6): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 19:54:56] php.WARNING: unlink(/home/sullivan/projects/ddn/news-connect/app/cache/de_/annotations/552d5470e454d5.25683391gJlEfl): No such file or directory {"type":2,"file":"/home/sullivan/projects/ddn/news-connect/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[...]
```

Indeed, `app/cache/de_`.

I don't have this error on previous version (2.7.x-dev 350f30b0e1651371a5a31c61c5d13c972b9d5c24)

**Comment by Soullivaneuh** [comment]
Just tested with a fresh symfony installation, same kind of results:

```bash
$ composer create-project symfony/framework-standard-edition test-sf 2.7.0-BETA1
[...]
$ cd test-sf
$ ./app/console cache:clear
Clearing the cache for the dev environment with debug true
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224e71ee7.458637660zQd3G): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224e7f5e5.17996454HHAppk): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224e8c362.74852656hpdiMX): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224e939c8.11129038sHAz9A): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224e9a0d5.74682371q5zcxe): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ea1090.19194179hLRcVR): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ea7101.95852581dzixjv): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ead851.90799179EQhdI8): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224eb4679.62982821mieg7L): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ebb469.27530446ymaGwp): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ec26c2.90867202CI1tW2): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ec9cf7.55562666youGmG): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ed0415.97750753FAseNj): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ed6699.657176958246dX): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224eddab5.40230827LbXnFA): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ee4755.59848014LYv16d): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224eeb8e2.24272889nQN2yR): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ef1a43.65343332Trmo1u): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224ef7f30.48595128WDZ4t8): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224efe7b4.53019055kIr7WL): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224f04911.75138695IFhuqp): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224f0a790.25801265oDQaU2): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:40] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d7224f11317.12729406wuSdoG): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
[2015-04-14 22:01:41] php.WARNING: unlink(/home/sullivan/tmp/test-sf/app/cache/de_/annotations/552d72250afb22.23602339xrYn4j): No such file or directory {"type":2,"file":"/home/sullivan/tmp/test-sf/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/FileCacheReader.php","line":214,"level":4352} 
```

**Comment by javiereguiluz** [comment]
I can reproduce the issue following the steps provided by @Soullivaneuh. The `cache:clear` command is the only one that fails, the rest of commands work as expected.

**Comment by fabpot** [comment]
fixed by symfony/symfony-standard#799

**Comment by xabbuh** [comment]
FYI, as @xelaris stated in https://github.com/symfony/symfony-standard/issues/798#issuecomment-93072930 this was an issue in the doctrine/annotations library. They published the `1.2.4` release last night which fixed our issue when bumping the version in the Symfony Standard Edition.
